src/components/patch/PatchContainer.tsx

```diff
@@ -1,20 +1,28 @@
-import { Grid, InputBase, Paper } from '@material-ui/core';
+import { Grid, InputBase, MenuItem, Paper, Select } from '@material-ui/core';
 import SearchIcon from '@material-ui/icons/Search';
-import { ConvertToPatches, UIVersion } from 'evergreen.js/lib/models';
+import { ConvertToPatches, UIPatch, UIVersion } from 'evergreen.js/lib/models';
 import * as React from 'react';
 import * as InfiniteScroll from 'react-infinite-scroller';
 import * as rest from "../../rest/interface";
 import '../../styles.css';
 import Banner from '../banner/Banner';
 import Patch from './Patch';
 
+enum SearchType {
```

Comment:
My intent was to have the project and status be discrete filters in addition to searching patch descriptions. I think the use cases will want an intersection of the filters

Reply:
Oh, I totally misunderstood the intent of this ticket. That's my bad, I'll go back now and start reworking my code.